samples/Utilities/Utilities.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.IO;
+
+namespace Common
```

Comment:
Instead of a Utiltites project, can you add this as a common file that is linked into each project (you can look at the pattern used by Directory.Build.props/targets in the src directory)

Reply:
No need to block this PR on that though